sourced/ml/transformers/repo2df.py

```diff
@@ -28,3 +28,24 @@ def process_row(self, row):
         for i, extractor in enumerate(self.extractors):
             for k in extractor.inspect(row.uast):
                 yield (i, k), 1
+
+
+class Repo2Quant(Transformer):
+
+    def __init__(self, extractors, nb_partitions, **kwargs):
+        super().__init__(**kwargs)
+        self.extractors = extractors
+        self.nb_partitions = nb_partitions
+
+    def __call__(self, rows):
+        for i, extractor in enumerate(self.extractors):
+            if "build_quantization" in dir(extractor):
```

Comment:
1. Better to rename to "quantize()". I saw that you already have that method - rename it too.
2.
```
try:
    quantize = extractor.quantize
except AttributeError:
    continue
# ...
```